hw/ip/aes/data/aes_testplan.hjson

```diff
@@ -47,8 +47,8 @@
     {
       name: back2back
       desc: '''
-           Randomly select the spacing between consecutive messages and blocks from 0 - n clock cycles.
-           The distribution will be weighted toward no and small gaps (0-10 cycles) but will also cover larger gaps.'''
+           Back to back blocks are not possible as the DUT need to be idle before writing a new configuration.
```

Comment:
This new description is not accurate:
- Back2Back blocks can very well be done. All that's needed is to read the STATUS register and check the INPUT_READY flag. This definitely needs to be tested and I would assume the current environment already covers this. Is that correct @rasmus-madsen ?
- Back2Back *messages* cannot be done as the DUT needs to be idle before writing a new configuration.

Reply:
I think I must have been asleep when I updated this text.
Back2back blocks are indeed verified.
I think all I wanted to say was that back to back messages are not possible as the configuration can not be changed while the DUT is Active

Reply:
This looks good, thanks for adjusting the description @rasmus-madsen !